kafka/adminapi_test.go

```diff
@@ -456,6 +458,38 @@ func testAdminAPIsListConsumerGroups(
 	if ctx.Err() != context.DeadlineExceeded {
 		t.Fatalf("Expected DeadlineExceeded, not %v", ctx.Err())
 	}
+
+	unknownGroupType, err := ConsumerGroupTypeFromString("Unknown")
+	if err != nil || unknownGroupType != ConsumerGroupTypeUnknown {
+		t.Fatalf("Expected ConsumerGroupTypeFromString to work for Unknown type")
+	}
+
+	listres, err := a.ListConsumerGroups(
+		ctx, SetAdminRequestTimeout(time.Second),
+		SetAdminMatchConsumerGroupTypes([]ConsumerGroupType{unknownGroupType}))
+	if err == nil {
+		t.Fatalf("Expected ListConsumerGroups to fail, but got result: %v, err: %v",
+			listres, err)
+	}
+	if ctx.Err() != context.DeadlineExceeded {
+		t.Fatalf("Expected DeadlineExceeded, not %v", ctx.Err())
+	}
+
+	classicGroupType, err := ConsumerGroupTypeFromString("Classic")
+	if err != nil || unknownGroupType != ConsumerGroupTypeUnknown {
+		t.Fatalf("Expected ConsumerGroupTypeFromString to work for Classic type")
+	}
+
+	listres, err := a.ListConsumerGroups(
+		ctx, SetAdminRequestTimeout(time.Second),
+		SetAdminMatchConsumerGroupTypes([]ConsumerGroupType{classicGroupType, classicGroupType}))
```

Comment:
```suggestion
		SetAdminMatchConsumerGroupTypes([]ConsumerGroupType{classicGroupType, classicGroupType}))
```
Why is the same group type included twice?

Reply:
It is to test the INVALID_ARGS error code with duplicate group types.

Reply:
But your test is not checking for invalid args error code below, so change that. You're changing for deadline exceeded still.